crates/holochain/src/conductor/api/api_cell.rs

```diff
@@ -84,10 +83,6 @@ impl CellConductorApiT for CellConductorApi {
         self.conductor_handle.keystore()
     }
 
-    fn signal_broadcaster(&self) -> SignalBroadcaster {
```

Comment:
All code related to the `SignalBroadcaster` has been replaced by the simpler `AppBroadcast`. The important difference is that the `AppBroadcast` is broadcasting on all connections to the app the is associated with the cell producing the signal, rather than all interfaces like the `SignalBroadcaster` did. That may be multiple interfaces but that possibility is not visible in conductor code anymore, it is contained within the logic of the `AppBroadcaster`.